### PR TITLE
Commit rector processed changes from CI 

### PR DESCRIPTION
**Pull request by JanMikes**

### CODE REVIEW

**Comment by JanMikes** [comment]
Work in progress, i just needed to trigger CI and it is only for pull requests now :-)

**Comment by JanMikes** [comment]
There is limitation that it can not run for PRs from forked repositories. It can be done via switching to personal access token, but i am not sure if it does not mean security risk. We can discuss tomorrow what are requested features :-)

**Comment by TomasVotruba** [comment]
Looks good :+1: 

Ready to merge for me

File: .github/workflows/rector_ci.yaml [hunk 1]
```diff
@@ -10,11 +10,31 @@ jobs:
...
+                name: Check for modified files
+                id: git-check
+                run: echo ::set-output name=modified::$(if git diff --exit-code -s; then echo "false"; else echo "true"; fi)
+
+            -   name: Push changes
+                if: steps.git-check.outputs.modified == 'true'
+                run: |
+                    git config --global user.name 'rector-bot'
+                    git config --global user.email 'tomas@getrector.org'
+                    git commit -am "Bot commit"
```
Comment (JanMikes) [inline .github/workflows/rector_ci.yaml]: @TomasVotruba i would love to improve this commit message, please do you have any suggestions?

File: src/DogFood/DogFoodClass.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Core\DogFood;
+
+final class DogFoodClass
```
Comment (JanMikes) [inline src/DogFood/DogFoodClass.php]: We can delete this class now, right?

Comment (JanMikes) [reply src/DogFood/DogFoodClass.php]: (will do it from the forked PR)

**Pull request merged**